### DISCUSSION TITLE
Generation of Discord Bot Tokens

### DISCUSSION THREAD

**Discussion by Nyzio2197** [community/community#10752] [Copilot Conversations]
I was writing a new class in Java titled Discord, and the first thing to pop up the following:
![image](https://user-images.githubusercontent.com/39226151/151687755-7870de4e-8b82-4772-aab4-aee6f5cc85bb.png)
I'm not sure if the token generated is a valid one or not, so I've put a censor on it just in case.

Shouldn't copilot assign either a null or an empty string instead of potential tokens?

**Comment by ShayBox** [comment]
Discord tokens are automatically invalidated if they're put on github or gist

**Comment by KTibow** [comment]
> Shouldn't copilot assign either a null or an empty string instead of potential tokens?

Remember, Copilot is a frontend to OpenAI Codex, which was trained from public GitHub repos. In general, it usually makes fake IDs. Given that it's AI, it can't really be changed on that side.
